src/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { Form } from '@bpmn-io/form-js';
+
+/**
+ * @typedef { import('@bpmn-io/form-js/dist/types').Data} Data
+ * @typedef { import('@bpmn-io/form-js/dist/types').Errors} Errors
+ * @typedef { import('@bpmn-io/form-js/dist/types').Schema} Schema
+ */
+
+/**
+ * @component
+ */
+export class ReactForm extends React.Component {
+
+  /**
+   * @param {Object} props
+   * @param {Data} props.data
+   * @param {Schema} props.schema
+   * @param {(error: Error) => {}} props.onError
+   * @param {(warnings: Array<any>) => {}} props.onShown
+   * @param {(result: { data: Data, errors: Errors }) => {}} props.onSubmit
+   * @param {() => {}} props.onReset
+   */
+  constructor(props) {
+    super(props);
+
+    this.containerRef = React.createRef();
+  }
```

Comment:
If you're going to keep using classes I think our ES version already support instance fields:

```suggestion
  containerRef = React.createRef();
```